### PR TITLE
feat/refactor imageupload and change tilelayer

### PR DESCRIPTION
**Pull request by lsiebels**
- For a better representation of the map, the [Static Tiles API](https://docs.mapbox.com/api/maps/static-tiles/?utm_source=mapboxcom&utm_medium=pricing#static-tiles-api-pricing) with the satellite style of Mapbox is now used. 
❗❗❗ Interaction with the basic maps is disabled except for zooming with the buttons to reduce the number of tiles that are loaded. (must be optimised) ❗❗❗
- Improved the placement of the previously cropped image. The image can now be arranged more easily because of the satellite style and the transparency of the image can be adjusted to provide the most accurate manual placement possible.
2022-06-27_13h49_45 
-  First revision of the Basic CSS layout. 
❗ This needs to be worked on in a separate issue, because the pages are still not really responsive.

### CODE REVIEW

**Comment by jarkenau** [comment]
As a suggestion, I would a fourth point, to mark all corner of the images. Probably doesn't add a lot of precision, but seems more intuitive for a user. Could definitely be implemented later.

**Comment by lucaslootalot** [comment]
@jarkenau That is a good point. Would make sense that if the image is not directly from the top, so the user can adjust the image better. I will create an issue for that.

**Comment by lsiebels** [comment]
@jarkenau created naturerobots/digital_logbook#23 🧐

File: vue/src/components/leaflet/LeafletComp.vue [hunk 1]
```diff
@@ -36,16 +36,39 @@ defineExpose({
...
+
+  //Disable map interaction, can be removed later but for now it reduces the number of tiles that need to be downloaded.
+  leafletMap.dragging.disable();
+  leafletMap.touchZoom.disable();
+  leafletMap.doubleClickZoom.disable();
+  leafletMap.scrollWheelZoom.disable();
+  leafletMap.boxZoom.disable();
+  leafletMap.keyboard.disable();
+  if (leafletMap.tap) leafletMap.tap.disable();
+
```
Comment (dobro929) [inline vue/src/components/leaflet/LeafletComp.vue:62]: removed **map options**
```suggestion
```

File: vue/src/components/leaflet/LeafletComp.vue [hunk 2]
```diff
@@ -36,16 +36,39 @@ defineExpose({
...
 
-  L.tileLayer("https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png", {
-    maxNativeZoom: 18,
-    maxZoom: props.maxZoom,
-  }).addTo(leafletMap);
+  //https://stackoverflow.com/a/55767702
+  if (props.zoomControl) {
+    L.control
+      .zoom({
+        position: "bottomright",
```
Comment (dobro929) [inline vue/src/components/leaflet/LeafletComp.vue:49]: shouldn't we rather use **topright**?

grafik

**Review by dobro929 — CHANGES_REQUESTED** [review]
For the revision of the layout I created the issue #154 👍🏼

File: vue/src/components/leaflet/LeafletComp.vue (hunk 2 above)
Comment (lucaslootalot) [reply vue/src/components/leaflet/LeafletComp.vue:49]: You are absolutely right. 👍🏻

**Review by dobro929 — APPROVED** [review]

**Pull request merged**